modules/flowable-engine/src/test/java/org/flowable/engine/test/bpmn/servicetask/RepeatingServiceTaskTest.java

```diff
@@ -13,21 +13,39 @@
 
 package org.flowable.engine.test.bpmn.servicetask;
 
+import org.flowable.common.engine.impl.history.HistoryLevel;
+import org.flowable.common.engine.impl.util.CollectionUtil;
+import org.flowable.engine.impl.test.HistoryTestHelper;
 import org.flowable.engine.impl.test.PluggableFlowableTestCase;
+import org.flowable.engine.runtime.ProcessInstance;
+import org.flowable.engine.test.Deployment;
+import org.flowable.variable.api.history.HistoricVariableInstance;
 import org.junit.jupiter.api.Test;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Please move all static imports to the top of the file under the package statement to comply with project standards; thanks.  To see the ordering of imports please see this ide settings f[ile](https://github.com/flowable/flowable-engine/blob/eec4e43e184c46723e354999622965619686db73/ide-settings/idea/idea-code-style-configuration.xml#L6).

Reply:
Thank you for your feedback, it has been modified.